style.css

```diff
@@ -1,7 +1,7 @@
 :root {
```

Comment:
### Your Boxes
![image](https://user-images.githubusercontent.com/93039359/194528006-04e1ffa6-4a94-43f4-ab4d-5d0e8a5a37c1.png)

### Figma Boxes
![image](https://user-images.githubusercontent.com/93039359/194528143-6688c01f-b8d5-4d6e-bdba-46210e5b629e.png)

- [ ] Your boxes are not equal size as your can compare it with the Figma template :blush: and make it look like Figma template :+1: